modules/drivers/velodyne/parser/velodyne32_parser.cc

```diff
@@ -98,14 +136,10 @@ void Velodyne32Parser::UnpackVLP32C(const VelodynePacket& pkt,
       raw_distance.bytes[0] = raw->blocks[i].data[k];
       raw_distance.bytes[1] = raw->blocks[i].data[k + 1];
 
-      // compute time
-      uint64_t timestamp = static_cast<uint64_t>(GetTimestamp(
-          basetime, (*inner_time_)[i][laser_id], static_cast<uint16_t>(i)));
-
-      if (laser_id == SCANS_PER_BLOCK - 1) {
-        // set header stamp before organize the point cloud
-        pc->set_measurement_time(static_cast<double>(timestamp) / 1e9);
-      }
+      // compute the actual timestamp associated with the data point at data
+      // block i and laser_id
+      uint64_t timestamp = GetTimestamp(basetime, (*inner_time_)[i][laser_id],
+                                        static_cast<uint16_t>(i));
```

Comment:
Please reformat the code here.

Reply:
I formatted the code with apollo/.clang-format. Anything specific here is not following the coding format requirements?